src/services/redis.service.js

```
@@ -0,0 +1,396 @@
+    try {
+      await client.quit();
+      console.log('Đã đóng kết nối Redis một cách an toàn');
+    } catch (error) {
+      console.error('Lỗi khi đóng kết nối Redis:', error);
+    }
+  }
+};
+
+/**
+ * @name isConnected
+ * @description Kiểm tra xem service đã kết nối đến Redis chưa
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)